routes/professions.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const pool = require('../connection/connection');
+
+const router = express.Router();
+
+// Get Method
+router.get('/:id', (req, res) => {
+    pool.getConnection((err, connection) => {
+        connection.query('', (error, results) => {
+            connection.release();
+            if (error) throw error;
+            res.send(results);
+        });
+        if (err) console.error(`Error with connection: ${err.message}`);
+    });
+});
+
+// Post Method
+router.post('/:id', (req, res) => {
```

Comment:
You don't want the `/:id` here since the database `id` property is `AUTO_INCREMENT` so we don't need to provide the id at all. Same with other files 😄